package.json

```diff
@@ -35,6 +35,7 @@
     "@types/chai": "^4.1.7",
     "@types/mocha": "^5.2.5",
     "@types/node": "^10.12.24",
+    "@types/pouchdb": "^6.3.3",
```

Comment:
Do we actually need to pull this in? We already have pouch working with only the `-core` import below.

Reply:
@types/pouchdb is needed to get typedefs for `import PouchDB from 'pouchdb';` We also have `import PouchDbMemory from 'pouchdb-adapter-memory';` which I believe uses the types from @types/pouchdb-core, but that package doesn't include everything we need.